lib/sms.rb

```diff
@@ -0,0 +1,35 @@
+require 'twilio-ruby'
+
+class Sms
+
+
+
+  def initialize(client_klass = Twilio::REST::Client)
+    @client_klass = client_klass
+    @client = get_client
+
+    @from = +441740582013
+    @to = +447753459230
+  end
+
+  def send(msg)
+    from = "+441740582013"
+    to = "+447753459230"
+    @client.account.messages.create(
+          :from => from,
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)